pkgs/development/libraries/drogon/default.nix

```diff
@@ -1,4 +1,12 @@
-{ stdenv, fetchFromGitHub, cmake, jsoncpp, libossp_uuid, zlib, openssl, lib }:
+{ stdenv, fetchFromGitHub, cmake, jsoncpp, libossp_uuid, zlib, openssl, lib,
+# miscellaneous
+brotliSupport ? true, brotli,
+c-aresSupport ? true, c-ares,
+# databases
+sqliteSupport ? true, sqlite,
+postgresSupport ? false, postgresql,
+redisSupport ? false, hiredis,
+mysqlSupport ? false, libmysqlclient, mariadb }:
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, cmake, jsoncpp, libossp_uuid, zlib, openssl, lib
# miscellaneous
, brotli, c-ares
# databases
, sqliteSupport ? true, sqlite
, postgresSupport ? false, postgresql
, redisSupport ? false, hiredis
, mysqlSupport ? false, libmysqlclient, mariadb }:
```

Reply:
Okay, I'll compromise here.  I don't see the harm in adding additional options though since (to my knowledge) hydra only caches the default options.

Reply:
Exactly. This is also the problem if most people need support for a backend. They then need to build the package.